doc/manuals/orion-api.md

```diff
@@ -995,20 +1131,85 @@ Content-Length: 31
 The temperature is 23.4 degrees
 ```
 
-Some considerations to take into account:
+If `payload` is set to `null`, then the notifications associated to that subscription will not 
+include any payload (i.e. content-length 0 notifications). Note this is not the same than using 
+`payload` set to `""` or omitting the field. In that case, the notification will be sent using 
+the NGSIv2 normalized format.
+
+As alternative to `payload` field in `httpCustom` or `mqttCustom`, the `json` field can be
+used to generate JSON-based payloads. For instance:
 
-* It is the NGSIv2 client's responsability to ensure that after substitution, the notification is a
+```
+"httpCustom": {
+   ...
+   "json": {
+     "t": "${temperature}",
+     "h": [ "${humidityMin}", "${humidityMax}" ],
+     "v": 4
+   }
+}
+```
+
+Some notes to take into account when using `json` instead of `payload`:
+
+* The value of the `json` field must be an array or object. Although a simple string or number is
+  also a valid JSON, these cases are not supported.
+* The macro replacement logic works the same way than in `payload` case, with the following
+  considerations:
+  * It cannot be used in the key part of JSON objects, i.e. `"${key}": 10` will not work
+  * The value of the JSON object or JSON array item in which the macro is used has to match
+    exactly with the macro expression. Thus, `"t": "${temperature}"` works, but
+    `"t": "the temperature is ${temperature}"` or `"h": "humidity ranges from ${humidityMin} to ${humidityMax}"`
+    will not work
+  * It takes into account the nature of the attribute value to be replaced. For instance,
+    `"t": "${temperature}"` resolves to `"t": 10` if temperature attribute is a number or to
+    `"t": "10"` if `temperature` attribute is a string.
+  * If the attribute doesn't exist in the entity, then `null` value is used
+* URL automatic decoding applied to `payload` and `headers` fields (described
+  [here](forbidden_characters.md#custom-payload-and-headers-special-treatment)) is not applied
+  to `json` field.
+* `payload` and `json` cannot be used at the same time
+* `Content-Type` header is set to `application/json`, except if overwritten by `headers` field
+
+Some considerations to take into account when using custom notifications:
+
+* It is the NGSIv2 client's responsibility to ensure that after substitution, the notification is a
   correct HTTP message (e.g. if the Content-Type header is application/xml, then the payload must
   correspond to a well-formed XML document). Specifically, if the resulting URL after applying the
   template is malformed, then no notification is sent.
 * In case the data to notify contains more than one entity, a separate notification (HTTP message)
   is sent for each of the entities (contrary to default behaviour, which is to send all entities in
   the same HTTP message).
+* Due to forbidden characters restriction, Orion applies an extra decoding step to outgoing
```

Comment:
Similar to https://github.com/telefonicaid/fiware-orion/pull/4181#discussion_r935481280

Probably after moving forbidden_characters.md#custom-payload-special-treatment here, the forbidden_characters.md get empty and can be removed (also from mkdocs.yml).